lib/commands/implementations/gulag.js

```diff
@@ -18,17 +20,48 @@ function gulag(defaultBanTime) {
       muteString,
       users: parsedUsers,
     } = services.gulag.parseInput(input, defaultBanTime);
+
     if (parsedUsers.length > 5) {
       return new CommandOutput(null, 'Too many users to be sent to the gulag, max of 5.');
     }
 
+    const users = _.uniq(
+      parsedUsers.map((name) => {
+        if (name.toLowerCase() === 'random') {
+          return services.roleCache.getRecentRandomUsername();
+        }
+        return name;
+      }),
+    );
```

Comment:
Isn't there a bug here where you can get the same random user multiple times and wind up with fewer users in the gulag than expected? I think it existed prior to your PR, so you can just slap it with a FIXME.